app/services/bookings/gitis/fake_crm.rb

```diff
@@ -18,6 +19,15 @@ def find_by_email(address)
       end
     end
 
+    def find_contact_for_signin(email:, firstname:, lastname:, date_of_birth:)
+      return super unless stubbed?
+      return nil if email =~ /unknown/
```

Comment:
Is this match a bit loose? The word `unknown` at any point in the local-part or domain or name would result in `nil` every time

Reply:
Never mind, ignore that - forgot I was looking at the fake crm!